### DISCUSSION TITLE
Visualization of the atomic configurations sampled by active learning

### DISCUSSION THREAD

**Discussion by LiangMD-BGI** [deepmodeling/dpgen#563] [Ideas]
Is there a possibility to include a tool in dpgen to visualize the configuration space sampled by active learning? An example can be seen in Figure 8 of the paper (https://doi.org/10.1038/s41467-021-21376-0). This would be helpful for the evaluation of the coverage of configuration space.

**Comment by AnguseZhang** [comment]
Thanks for your suggestions!  Indeed, visualization may help understand the configuration space.  However, the example you list (Figure 8) may be too specific. They did some dimension reduction, and different users' needs may vary significantly.  Do you think there are some general needs  that our implementation can be applied to different scenarios ?
